README.md

```diff
@@ -1,52 +1,192 @@
-## BEGIN - TO DELETE TILL END
+# reshuffle-shopify-connector
 
-THIS IS A TEMPLATE REPO FOR NEW RESHUFFLE CONNECTORS
-1. Create a new connector repo from this template using this link https://github.com/reshufflehq/reshuffle-template-connector/generate
-2. Clone the repo locally
-3. Rename all occurrences of _CONNECTOR_NAME_
-4. `npm install`
-5. `npm run build:watch`
-6. Implement your events/actions in `src/index.ts`
-7. `npm run lint`
-8. Push your code
-9. Go to https://app.circleci.com/projects/project-dashboard/github/reshufflehq/
-    a. You should see your new connector repo
-    b. click on `Set Up Project` for the repo
-    c. click on `Use Existing Config`
-    d. click on `Start Building`
+[Code](https://github.com/reshufflehq/reshuffle-shopify-connector) |  [npm](https://www.npmjs.com/package/reshuffle-shopify-connector) | [Code sample](https://github.com/reshufflehq/reshuffle/tree/master/examples/shopify)
 
-10. If circle CI checks are all green, you are all set!
+`npm install reshuffle-shopify-connector`
 
-// Keep documentation template below
+This connector uses [Official Node Shopify Client](https://www.npmjs.com/package/shopify-api-node) package.
 
-## END
+### Reshuffle Shopify Connector
 
-# reshuffle-_CONNECTOR_NAME_-connector
+This connector provides a connector for [Shopify](https://www.shopify.com/online).
 
-### Reshuffle _CONNECTOR_NAME_ Connector
+The following example listen to Shopify events:
+```js
+const { Reshuffle } = require('reshuffle')
+const { ShopifyConnector } = require('reshuffle-shopify-connector')
 
-This connector provides <description>.
+const app = new Reshuffle()
+const connector = new ShopifyConnector(app, {
+  shopName: process.env.SHOPIFY_SHOP_NAME,
+  apiKey: process.env.SHOPIFY_API_KEY,
+  password: process.env.SHOPIFY_PASSWORD
+})
+  
+connector.on({ topic: 'orders/fulfilled' }, (event, app) => {
+  console.log('congratulations! Another order fulfilled')
+})
+
+app.start()
+```
+
+Create an API token from your Shopify Admin Page:
+
+1. Go to https://<your_shop_name>.myshopify.com/admin/apps/private
+2. Click `Create new private app`, 
+3. Provide a name and scopes (keeping in mind that permissions are very limited per default)
+4. Click Save, and you'll get api key and password
 
 #### Configuration Options:
+
+Provide options as below for connecting to Shopify:
```

Comment:
Is this for if you only want to use the actions? If this is a valid config (which only means you don't get to have events) - then maybe we should point this out?